application/dash_application/known_pages_router.py

```diff
@@ -1,34 +1,92 @@
+from collections import namedtuple
+import dash_bootstrap_components as dbc
 import dash_html_components as html
 import dash_core_components as core
 from dash.dependencies import Input, Output
+
 from .dash_id import init_ids
 from . import pages
+from ..routes import version
 
 
 """
 This file acts as a router which serves all the pages the Dash app knows about.
 """
 page_name = None
+default_title = ""
+
+# Build dict of known pages info
+pages_info = {}
+PageInfo = namedtuple('PageInfo', 'layout title')
+for p in pages.pages:
+    pages_info[p.page_name] = PageInfo(p.layout, p.title)
+
+def navbar(current):
+    def menu_item(label, link):
+        return dbc.DropdownMenuItem(label,
+                                    href=link,
+                                    external_link=True,
+                                    disabled=current == label,
+                                    style={"fontSize": "12pt"})
+    return dbc.NavbarSimple(
+        id="navbar",
+        children=[
+            dbc.DropdownMenu(
+                children=[
+                    menu_item("Pre-Exome", "preqc-exome"),
```

Comment:
You could build this from `pages_info` if inclined.

Reply:
That's much more elegant.